tools/windows-node-installer/README.md

```diff
@@ -1,17 +1,17 @@
 # windows-node-installer
 The `windows-node-installer (wni)` is a tool that creates a Windows instance under the same virtual network 
 (AWS-VCP, Azure-Vnet, and etc.) used by a given OpenShift cluster running on the selected provider.
-The actual configuration on the created Windows instance is done by 
-[WMCB](https://github.com/openshift/windows-machine-config-operator) to ensure that the node joins the
-OpenShift cluster
+The actual configuration on the created Windows instance is done by the 
+[WMCB](https://github.com/openshift/windows-machine-config-operator) to ensure that the instance joins the
+OpenShift cluster as a Windows worker node.
 
 ### Supported Platforms
  
  - AWS
  
 ### Pre-requisite
 
- - An existing OpenShift 4.x cluster running on a supported platform.
+ - An existing OpenShift 4.2.x cluster running on a supported platform.
```

Comment:
I think this should be 4.x, if we've issues with 4.1 then we need to call it out but saying we're going to support 4.2.x doesn't solve our purpose :)

Reply:
I am not sure why it should be 4.x given we definitely are not supporting anything prior to 4.2.

Reply:
I specifically mention 4.2.x because it does not support 4.1. There is one minor change from 4.1 -> 4.2 which platform is deprecated. https://github.com/openshift/api/blob/d752fe3c256069e22042e626f0eed375c9b9760b/config/v1/types_infrastructure.go#L44 Therefore, the code will not work on 4.1 @ravisantoshgudimetla @aravindhp

Reply:
I say we leave it as 4.2.x